app/players/elves/elf.py

```diff
@@ -0,0 +1,14 @@
+from abc import ABC
+from app.players.player import Player
+
+
+class Elf(Player, ABC):

```

Comment:
Forgotten `ABC` redundat class )) Try always check if you fixed mistake in all places : )